nix/difftest/riscv-tests.nix

```diff
@@ -0,0 +1,20 @@
+{ pkgsCross, fetchFromGitHub }:
```

Comment:
I think we should make a `rv32-stdenv` alias at overlay.nix, and use it as argument. This could help easier override.